package.json

```diff
@@ -15,6 +15,7 @@
     "classnames": "^2.2.5",
     "express": "^4.16.2",
     "express-uncapitalize": "^1.0.0",
+    "https": "^1.0.0",
```

Comment:
@dmytropanontko , could we just use regular [fetch](https://developer.mozilla.org/ru/docs/Web/API/Fetch_API/Using_Fetch) for our purposes?

Reply:
@epaminond I think yes, but need to check